ClosedXML/ClosedXML.csproj

```diff
@@ -3,22 +3,37 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net40;net46</TargetFrameworks>
     <LangVersion>7.2</LangVersion>
+    <AssemblyName>ClosedXML</AssemblyName>
+    <PackageId>ClosedXML</PackageId>
     <Version>0.93.0</Version>
-    <Authors>Manuel de Leon, Amir Ghezelbash, Francois Botha</Authors>
+    <Authors>Manuel de Leon, Amir Ghezelbash, Francois Botha, Aleksei Pankratev</Authors>
     <Company />
-    <Product />
+    <Product>ClosedXML</Product>
+    <PackageReleaseNotes>See https://github.com/ClosedXML/ClosedXML/releases/tag/$(productVersion)</PackageReleaseNotes>
     <Description>ClosedXML makes it easier for developers to create Excel 2007+ (.xlsx, .xlsm, etc) files. It provides a nice object oriented way to manipulate the files (similar to VBA) without dealing with the hassles of XML Documents. It can be used by any .NET language like C# and VisualBasic.NET.</Description>
     <Copyright>MIT</Copyright>
     <PackageLicenseUrl>https://github.com/ClosedXML/ClosedXML/blob/master/LICENSE</PackageLicenseUrl>
     <PackageProjectUrl>https://github.com/ClosedXML/ClosedXML</PackageProjectUrl>
     <RepositoryUrl>https://github.com/ClosedXML/ClosedXML</RepositoryUrl>
-    <SignAssembly>true</SignAssembly>
-    <AssemblyOriginatorKeyFile>ClosedXML.snk</AssemblyOriginatorKeyFile>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <NoWarn>$(NoWarn);NU1605;CS1591</NoWarn>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <Configurations>Debug;Release;Release.Signed</Configurations>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)'=='Release.Signed'">
+    <OutputPath>bin\Release.Signed\</OutputPath>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <SignAssembly>true</SignAssembly>
+    <Optimize>true</Optimize>
+    <AssemblyOriginatorKeyFile>ClosedXML.snk</AssemblyOriginatorKeyFile>
+    <DefineConstants>$(DefineConstants);RELEASE;STRONGNAME</DefineConstants>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)'=='Release'">
```

Comment:
Does this add dependencies automatically?

Reply:
Yes, dependencies are added automatically (you can check .nupkg file in build artifacts). So we don't have to bother keeping .nuspec synchronized with .csproj.